docs/com/api/jsonata4java/expressions/utils/BooleanUtils.html

```diff
@@ -201,16 +201,32 @@ <h3>Method Detail</h3>
 <li class="blockList">
 <h4>convertJsonNodeToBoolean</h4>
 <pre>public static&nbsp;boolean&nbsp;convertJsonNodeToBoolean(com.fasterxml.jackson.databind.JsonNode&nbsp;node)</pre>
-<div class="block">See http://docs.jsonata.org/boolean-functions.html#booleanarg
- 
- The convertJsonNodeToBoolean method converts the node passed in to a boolean
- based on the casting semnatics defined by JSONata.
```

Comment:
i'm not quite sure where this should go. Since it sorta fell in the middle of a block i'm consistently replacing... I kicked it below, but i'm not wed to that.